utils/checks.stanza

```diff
@@ -440,25 +439,25 @@ defn check-a-pull-resistor-exists (open-collectors: Tuple<DigitalPin>, pull-up-r
     condition =            not empty?(pull-up-resistors),
     name =                 "IO pin checks"
     description =          "Check io pin properties"
-    category =             "pin-check"
+    category =             "Pin Checks"
     subcheck-description = "Check that io pins marked OpenCollector have at least one pull-up resistor",
     pass-message =         open-collector-pass-message(open-collectors, "at least one pull-up resistor"),
     fail-message =         open-collector-fail-message(open-collectors, "any pull-up resistor attached"),
     locators =             to-jitx-pins(open-collectors))
 
-defn check-and-return-unique-rail-voltage (rail-voltages: Tuple<Double>, pull-up-resistors: Tuple<Instance>, open-collectors: Tuple<DigitalPin>) -> Double :
+defn check-and-return-unique-net-voltage (net-voltages: Tuple<Toleranced>, pull-up-resistors: Tuple<Instance>, open-collectors: Tuple<DigitalPin>) -> Toleranced :
   #CHECK(
-    condition =            all-equal?(rail-voltages),
+    condition =            all-equal?(net-voltages),
     name =                 "IO pin checks"
     description =          "Check io pin properties"
-    category =             "pin-check"
+    category =             "Pin Checks"
     subcheck-description = "Check that io pins marked OpenCollector have pull-up resistors connected to the same rail voltage",
     pass-message =         open-collector-pass-message(open-collectors, "all pull-up resistors with the same rail voltage"),
```

Comment:
Messages still have the term `rail-voltage`, but can we still have the same logic like this? pull-up resistors need to be attached to supply rails.